Automattic/wp-calypso
# Marketplace: Adds Yoast spotlight banner to featured plugins

#### Changes proposed in this Pull Request
* Adds the `Spotlight` component that can be reused. (Maybe should be called `SpotlightBanner`?).
* Adds the banner to the plugin page hidden behind the `marketplace-spotlight` feature flag.

#### Testing instructions
* Navigate into `http://calypso.localhost:3000/plugins/<siteId>?flags=marketplace-spotlight`
* Spotlight should appear below the title in the **Featured** section.
* Navigate into `http://calypso.localhost:3000/plugins/<siteId>`
* Spotlight should not appear.

#### Screenshots
|  Large Screen | Mobile Screen   |  
|---------------|-----------------|
|![Screen Shot 2022-02-18 at 5 05 54 PM](https://user-images.githubusercontent.com/1035546/154767711-86d71436-9404-4022-a4d1-614ed2c3423c.png)|![Screen Shot 2022-02-18 at 5 06 11 PM](https://user-images.githubusercontent.com/1035546/154767776-ab02e051-784a-4fa3-9173-e65cdf586207.png)|

Related to #60923
